src/main/resources/assets/jei/lang/de_DE.lang

```diff
@@ -1,64 +1,88 @@
 # Tooltips
-jei.tooltip.config=Konfiguration
+jei.tooltip.config=JEI-Konfiguration
+jei.tooltip.show.recipes=Zeige Rezepte
 jei.tooltip.show.all.recipes=Zeige alle Rezepte
 jei.tooltip.delete.item=Klicke zum Löschen
 jei.tooltip.liquid.amount.with.capacity=%,d / %,d mB
 jei.tooltip.liquid.amount=%,d mB
+jei.tooltip.transfer=Bewege Gegenstände
+jei.tooltip.recipe.ore.dict=Akzeptiert: %s
+jei.tooltip.item.colors=Farben: %s
+jei.tooltip.shapeless.recipe=Formloses Rezept
+jei.tooltip.cheat.mode=Cheat-Modus
 
 # Error Tooltips
-jei.tooltip.error.recipe.transfer.missing=Fehlende Items
-jei.tooltip.error.recipe.transfer.inventory.full=Inventar ist voll
+jei.tooltip.error.recipe.transfer.missing=Fehlende Gegenstände
```

Comment:
Well, this sounds like a fun discussion - let me take part in it :)

1) Modern languages are evolving. That's just how languages work. The people that use it, also adjust it to fit the environment they live in. If you don't like this, stick to a dead language.

2) I would also consider "Item" to be a perfectly good choice in this context. There even is a [Wikipedia page](https://de.wikipedia.org/wiki/Item_(Computerspiel)) to explain the meaning of the word to those who don't understand it.

3) Having consistent naming is important. I've seen many players get confused by inconsistent naming (i.e. "fluid" vs. "liquid" and "pipe" vs. "tube").

4) Can we just keep "Item"? At least until Mojang fixes the GUI system?